Makefile

```diff
@@ -45,7 +45,7 @@ INSTALL=install
 # checkout. Invoke make with GOOGLE_PROTOBUF_CHECKOUT=[PATH_TO_CHECKOUT] to
 # override this value, i.e. -
 #   make [TARGET] GOOGLE_PROTOBUF_CHECKOUT=[PATH_TO_CHECKOUT]
-GOOGLE_PROTOBUF_CHECKOUT=../protobuf
+GOOGLE_PROTOBUF_CHECKOUT?=../protobuf
```

Comment:
The GNU Make manual makes a good point that you should be cautious about accepting settings from the environment, so I'm not in a hurry to do this globally.